pages/pipelines/_block_wait.md.erb

```diff
@@ -0,0 +1,34 @@
+## Block steps interacting with wait steps
+
+If a block step follows or precedes a wait step in your build, the wait step will be ignored and only the block step will run, like this example:
+
+```yml
+steps:
+  - command: ".buildkite/steps/yarn"
+  - wait: ~
+  - block: "unblock me"
+```
+{: codeblock-file="pipeline.yml"}
+
+But let's consider a different example. Now the wait step will be ignored, but the block step will not be executed because `continue_on_failure` is set to `true`.
```

Comment:
```suggestion
But let's consider a different example. Now the wait step (with `continue_on_failure` is set to `true) will be ignored, but the block step will also **not run**, because the 'previous' command step failed.
```

Reply:
@nithyaasworld thanks for the detailed explanation. It look me a surprisingly long time to understand what was going on.

I **love** that you've thought about documenting a common anti-pattern, and explaining it. Just double check that my rewording makes sense as you understand it, and we can hit the merge button.

Reply:
Its looks good @plaindocs 🚀. Thanks for checking!